src/test/java/com/hongliangjie/fugue/topicmodeling/LDA/LDATest.java

```diff
@@ -0,0 +1,177 @@
+            LOGGER.info("MAX_ITER:" + MAX_ITER);
+
+            Sampler g1 = new GibbsSampling();
+            Sampler g2 = new GibbsLogSampling();
+            ProcessDocuments p = new TestProcesser(g1,g2);
+            g1.setProcessor(p);
+            g2.setProcessor(p);
+            p.sampleOverDocs(internalDocs, MAX_ITER, 0);
+        }
+
+        public void countsCheck() {
+            int topic_sum = 0;
+
+            for (int k = 0; k < TOPIC_NUM; k++) {
+                topic_sum += topicCounts[k];
+            }
+
+            assertEquals("topicCounts are not consistent with total token:", topic_sum, TOTAL_TOKEN);
+
+            int term_topic_sum = 0;
+            int[] topic_sums = new int[TOPIC_NUM];
+            for (int v = 0; v < wordTopicCounts.size(); v++) {
+                int[] c = wordTopicCounts.get(v);
+                for (int k = 0; k < c.length; k++) {
+                    term_topic_sum += c[k];
+                    topic_sums[k] += c[k];
+                }
+            }
+
+           assertEquals("wordTopicCounts are not consistent with total token:", term_topic_sum, TOTAL_TOKEN);
+
+
+            for (int k = 0; k < TOPIC_NUM; k++) {
+                assertEquals("topic_sums are not consistent with topicCounts:" + k, topic_sums[k], topicCounts[k]);
+            }
+
+            int[] document_agg_topics = new int[TOPIC_NUM];
+            for (int d = 0; d < internalDocs.size(); d++) {
+                List<Integer> docTopicAssignment = docTopicAssignments.get(d);
+                int[] docTopicBuffer = docTopicBuffers.get(d);
+                int pos = internalDocs.get(d).getFeatures().size();
+                int total_topic_sum = 0;
+                for (int k = 0; k < TOPIC_NUM; k++) {
+                    total_topic_sum += docTopicBuffer[k];
+                }
+                assertEquals("The total_topic_sum is not consistent with number of tokens:" + d, total_topic_sum, pos);
+
+                for (Integer z : docTopicAssignment) {
+                    document_agg_topics[z] += 1;
+                }
+
+                assertEquals("The total_tokens is not consistent with number of tokens:" + d, docTopicAssignment.size(), pos);
+            }
+
+            for (int k = 0; k < TOPIC_NUM; k++) {
+                assertEquals("The aggregated topic assignment is not consistent with topicCounts:" + k, document_agg_topics[k], topicCounts[k]);
+            }
+            LOGGER.info("Counts Verifized,");
+        }
+    }
+
+    private Message prepareDocuments() throws Exception{
+        DataReader r = new DataReader();
+        Message msg = new Message();
+        URL testFileURL = this.getClass().getClassLoader().getResource("ap-test.json");
+        msg.setParam("inputFile", testFileURL.getPath());
+        msg.setParam("topk", 100);
+        msg = r.Read(msg);
+        return msg;
+    }
+
+
+    @Test
+    public void testTrain() throws Exception {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://www.codacy.com/app/hongliangjie/Fugue/file/2178366275/issues/source?bid=3162994&fileBranchId=3163615#l168)